### ISSUE TITLE
Debugadapter protocol: later configuration of exception breakpoints

### ISSUE THREAD

**Issue by maziac** [microsoft/vscode#169474]
Hi,

the debug adapter initialization sequence is (simplified):
1.a initializeRequest
1.b response: capabilities
3. setBreakpointsRequest
4. launchRequest with the SettingsParameters (i.e. the contents from the launch.json)

In my implementation of the debug adapter I have the problem that the debug adapter has to communicate with an external emulator (after step 4).
From the emulator some capabilities are retrieved.
The debug adapter can talk to several emulators. The launch.json tells which one to use.

Here comes the problem: When returning the capabilities in step 1.b it is not known yet what emulator should be used and what/if exception breakpoints are supported.
This is known after step 4. But then there is no way configure exception breakpoints anymore.

So here is my feature request:
Instead of configuring exception breakpoints in step 1, move it to step 4, e.g. to the response of the launchRequest.
Or, alternatively, add a new command that would allow configuration of the exception breakpoints dynamically.

**Comment by roblourens** [comment]
Can you use a `CapabilitiesEvent` to change them later?

**Comment by maziac** [comment]
I have used CapabilitiesEvent in the past and changed e.g. the 'supportsStepBack' property with success.

But for the exception breakpoints this does not seem to work.

**Comment by roblourens** [comment]
It seems that we don't update exception breakpoints for a `CapabilitiesEvent`

**Comment by maziac** [comment]
Will you change/fix this behavior?

**Comment by roblourens** [comment]
cc @nisargjhaveri just in case you feel like working on this

**Comment by maziac** [comment]
Hi, is there any progress on this?

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
